### PR TITLE
feat: fix CI for macos-latest

### PR DESCRIPTION
**Pull request by chenrui333**
fix CI pipeline failures, seeing:

```
/Users/runner/work/_temp/b78b4be8-0b82-444d-b150-be4b53a8afc4.sh: line 19: initdb: command not found
```

relates to https://github.com/ikalnytskyi/action-setup-postgres/pull/29

### CODE REVIEW

**Comment by jiacai2050** [comment]
看上游issue ，14不是已经支持了吗？

**Comment by chenrui333** [comment]
14, 13 都支持 I will followup on that

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -42,7 +42,7 @@ jobs:
         with:
           character set server: 'utf8'
           collation server: 'utf8_general_ci'
-          mysql version: '8.0'
+          mysql version: '8.3'
```
Comment (chenrui333) [inline .github/workflows/ci.yml:45]: this is to match with mysql-client brew dependency

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -57,15 +57,24 @@ jobs:
         if: matrix.os == 'macos-latest'
         run: |
           prefix=$(brew --prefix)
-          echo "PKG_CONFIG_PATH=${prefix}/opt/libpq/lib/pkgconfig:${prefix}/opt/mysql-client/lib/pkgconfig" >> ${GITHUB_ENV}
+          echo "PKG_CONFIG_PATH=${prefix}/opt/sqlite/lib/pkgconfig:${prefix}/opt/libpq/lib/pkgconfig:${prefix}/opt/mysql-client/lib/pkgconfig" >> ${GITHUB_ENV}
```
Comment (chenrui333) [inline .github/workflows/ci.yml:60]: add sqlite pkg-config, seeing in ci run

```
bash-3.2$ zig build -Dtarget=aarch64-macos
install
└─ install examples-14-01
   └─ zig build-exe examples-14-01 Debug aarch64-macos failure
error: error: unable to find dynamic system library 'sqlite3' using strategy 'paths_first'. searched paths: none

error: the following command exited with error code 1:
/Users/runner/hostedtoolcache/zig/master/arm64/zig build-exe -lsqlite3 -ODebug -target aarch64-macos -mcpu baseline --dep build-info -Mroot=/Users/runner/work/zig-cookbook/zig-cookbook/src/14-01.zig -Mbuild-info=/Users/runner/work/zig-cookbook/zig-cookbook/zig-cache/c/2a81331826b38a5d6b96b59574c1e7fb/options.zig -lc --cache-dir /Users/runner/work/zig-cookbook/zig-cookbook/zig-cache --global-cache-dir /Users/runner/.cache/zig --name examples-14-01 --listen=-
```

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -57,15 +57,24 @@ jobs:
...
           zig fmt --check src/
           zig build
           zig build run-all --summary all
 
+      - name: Run examples(macOS)
+        if: matrix.os == 'macos-latest'
+        run: |
+          pkg-config --libs --cflags libpq mysqlclient
+          zig fmt --check src/
+          zig build -Dtarget=aarch64-macos
```
[the remark this answers is not in the document]
Comment (chenrui333) [reply .github/workflows/ci.yml:75]: 需要加 target 是错的

**Review by jiacai2050 — APPROVED** [review]

**Pull request merged**